playbooks/azure/openshift-cluster/build_gi.yml

```diff
@@ -4,19 +4,26 @@
 # create all the items necessary to make the VM
 - name: "Create resource group"
   hosts: localhost
-  gather_facts: no
+  gather_facts: yes
   tasks:
   - name: "provision public ip address"
     import_role:
       name: openshift_azure
       tasks_from: provision_instance.yml
+  - name: get the oreg token
+    import_role:
+      name: openshift_azure
+      tasks_from: oreg_token.yml
```

Comment:
I'm not sure this can be placed here as the `oc` binary is not yet on the node.

Reply:
This runs from the host creating the infrastructure, so a workstation, or a container.